nav2_behavior_tree/include/nav2_behavior_tree/bt_service_node.hpp

```diff
@@ -142,6 +142,16 @@ class BtServiceNode : public BT::ActionNodeBase
   {
   }
 
+  /**
```

Comment:
Please add to migration guide a new section on the Galactic page that we added this virtual function to allow for setting of codes after services return

Reply:
Are you referring to this [file](https://github.com/ros-planning/navigation.ros.org/blob/1278dcc676a8ea2c1b9207351a49e68ccb50f1d2/migration/Foxy.rst) ?
Would the best way to edit it be to create a fork and than PR the changes ?

Reply:
https://github.com/ros-planning/navigation.ros.org/blob/master/migration/Galactic.rst this file